client/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.flipkart</groupId>
         <artifactId>flux</artifactId>
-        <version>1.1.4-SNAPSHOT</version>
+        <version>1.2.0-SNAPSHOT</version>
```

Comment:
Not part of this one, but delete the earlier 1.2.0-SNAPSHOT git tag, create again as the old code results in compilation errors

Reply:
still not merging in master, will create new tag from master, with correct version details.